module/VuFind/src/VuFind/Auth/ILSAuthenticator.php

```diff
@@ -151,7 +151,7 @@ public function encrypt(?string $text)
      *
      * @param ?string $text    The text to be encrypted or decrypted
      * @param bool    $encrypt True if we wish to encrypt text, False if we wish to
-     * decrypt text.
+     *                         decrypt text.
```

Comment:
It looks like some of the whitespace in the comments got adjusted unnecessarily. Is it possible your IDE has a setting that's getting a little carried away? ;-)

Reply:
My IDE definitely decided everything needed to be justified. I have corrected this.